src/controllers/expenses.controller.js

```diff
@@ -0,0 +1,86 @@
+const { expensesService } = require('../services/expense.services.js');
+const { usersService } = require('../services/user.services.js');
+
+const getExpenses = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+  const categoryList = categories ? categories.split(',') : [];
+
+  const expenses = expensesService.getExpenses({
+    userId,
+    categoryList,
+    from,
+    to,
+  });
+
+  res.json(expenses);
+};
+
+const createExpense = (req, res) => {
+  const { userId, spentAt, title, amount, category, note } = req.body;
+
+  if (!userId || !spentAt || !title || !amount || !category || !note) {
+    return res.sendStatus(400);
+  }
+
+  const user = usersService.getUserById(userId);
+
+  if (!user) {
+    return res.sendStatus(400);
```

Comment:
The error response for a non-existent user should include a descriptive message. Consider using `res.status(400).json({ message: 'User not found' });` to provide more context.